src/screens/login/components/OidcOnboardingView.tsx

```diff
@@ -0,0 +1,102 @@
+import Minilog from '@cozy/minilog'
+import React, { useEffect, useState } from 'react'
+import { KeyboardAvoidingView, Platform, StyleSheet, View } from 'react-native'
+
+import { SupervisedWebView } from '/components/webviews/SupervisedWebView'
+
+import { parseOidcOnboardingFinishedUrl } from './functions/oidc'
+
+import { NetService } from '/libs/services/NetService'
+import { routes } from '/constants/routes'
+
+const log = Minilog('OidcOnboardingView')
+
+interface ClouderyWebViewProps {
+  code: string
+  onboardUrl: string
```

Comment:
shouldn't be `onboardingUrl`?

Reply:
Here I chose to fit to the queryParam name from the Cloudery result. Here is an example url I get as the IAB result:
https://links.mycozy.cloud/flagship/oidc_result?code=f38714d48d39916deb2e4577&onboard_url=http%3A%2F%2Flocalhost%3A3000%2Fv2%2Fgrdlyon%2Fstart%3Fonboarding_id%3Du3dF9tYnNqIG763W